### DISCUSSION TITLE
Serialnumber at device is double

### DISCUSSION THREAD

**Discussion by mactoolz** [Louisvdw/dbus-serialbattery#982] [General]
Hello,

i use the seplos BMS at my Victron system. At each seplos BMS i have configured the parameter to see which is the right storage at the driver, i configured the capacity parameter with different value.

But the driver shows me two serialnumber twice the same seralnumber.
I checked all my storages and there are different capacities values.

how can i correct the results at the driver.

Thank you

**Comment by mr-manuel** [comment]
Please open a support issue, so I get all informations I need to help you.
